cluster/images/kubemark/Makefile

```diff
@@ -15,18 +15,17 @@
 # build Kubemark image from currently built binaries containing both 'real' master and Hollow Node.
 # This makefile assumes that the kubemark binary is present in this directory.
 
-REGISTRY?=gcr.io
-PROJECT?=google_containers
+REGISTRY ?= staging-k8s.gcr.io
```

Comment:
Looks like something is overriding this in testing:
```
I0206 21:23:43.775] gcloud docker -- push gcr.io/kubemark
I0206 21:23:44.496] The push refers to a repository [gcr.io/kubemark]
W0206 21:24:34.697] unknown: Pushing to root-level images is disabled. See https://cloud.google.com/container-registry/docs/support/deprecation-notices
W0206 21:24:34.760] make: *** [gcloudpush] Error 1
```

https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/57824/pull-kubernetes-kubemark-e2e-gce/75252/?log#log

Reply:
Maybe it's this: https://github.com/kubernetes/kubernetes/blob/ebffd76acf84d14d7d72fda747e98429d0932895/cluster/images/kubemark/BUILD#L15-L16

Reply:
Doesn't look like `docker_build` messes with env vars:
https://github.com/bazelbuild/rules_docker/blob/master/docker/docker.bzl
https://github.com/bazelbuild/rules_docker/blob/master/container/push.bzl

Reply:
adding here: it's not this, that job is using the makefile build. I've poked through test-infra a bit and I don't see this env being set for this job though...

Reply:
I know how to fix the makefile part here, I don't know how to fix the bazel part.  Pushing now.  PTAL at bazel equiv.

Reply:
pushed another, trying to get bazel.

Bazel doesn't seem to have the same semantic as make here.  Make allows "if this variable is not set, use a default".  So I just repeated the default value.  Sigh.  Someone who knows this script can maybe take a look later ( @shyamjvs )